src/gowrapper/ckks/ciphertext.go

```diff
@@ -34,6 +34,12 @@ func lattigo_ciphertextScale(ctHandle Handle8) float64 {
 	return ctIn.ScalingFactor()
 }
 
+//export lattigo_ciphertextDegree
+func lattigo_ciphertextDegree(ctHandle Handle8) int {
+	ct := getStoredCiphertext(ctHandle)
+	return ct.Degree()
```

Comment:
I see this function in [Lattigo HEAD](https://github.com/tuneinsight/lattigo/blob/master/rlwe/ciphertext.go#L50) but not in [Lattigo 2.2.0](https://github.com/tuneinsight/lattigo/blob/c629f0c9518f6117141f54e441d6ca5d765f8868/ckks/ciphertext.go), which is what this library is based currently uses. Where is this function defined in Lattigo v2.2.0?

Please make this return a `uint64`, as noted elsewhere.